tempsensor/makefile

```diff
@@ -0,0 +1,20 @@
+CC = g++
+LD = g++
+CCFLAGS = -Wall
+LDFLAGS = -lstdc++ -pthread -L ./target/debug/
+EXE = tempsensor

```

Comment:
call it `tempsensord`, that's daemon naming convention

Reply:
Fixed it in the most recent commit.